streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -384,9 +390,8 @@ public static StreamThread create(final InternalTopologyBuilder builder,
             builder,
             threadId,
             logContext,
-            assignmentErrorCode,
-            nextScheduledRebalanceMs
-        );
+            nextScheduledRebalanceMs,
+            assignmentErrorCode);
```

Comment:
The closing parenthesis should be in a new line.
```suggestion
            assignmentErrorCode
        );
```

Reply:
fixed